action.yml

```diff
@@ -1,36 +1,119 @@
-name: Integration Test in Tutor
-description: 'A Github action to test your plugin in Tutor (Open edX distribution)'
+name: Open edX Plugin Integration Tests with Tutor
+description: "A Github action to test your plugin in Tutor (Open edX distribution)"
 inputs:
   app_name:
-    description: 'Application name to test. E.g., eox-tenant.'
+    description: "Application name to test. E.g., eox-tenant."
     required: true
   tutor_version:
-    description: 'The tutor version matrix to use.'
+    description: "The tutor version matrix to use."
     required: true
   shell_file_to_run:
-    description: 'The path of the shell file to run the integration tests.'
+    description: "The path of the shell file to run the integration tests."
+    required: true
+  openedx_extra_pip_requeriments:
+    description: "Optional extra pip requirements to install in Open edX. E.g: 'package1==1.0 package2>=2.0'"
+    required: false
+    default: ""
+  python_version:
```

Comment:
In which cases would this input (`python_version`) be useful?

Reply:
I couldn't find a case where it would be useful. I removed the step because I think it's better practice to use the default Python version from the runner. There's no really a need to install a specific Python version

Thanks!